:fix: レスポンシブのUI修正

kei-nishikawa48 opened the pull request:
- レスポンシブ時tabでcodeかpreviewを切り替えらるように
- apikeyが空でもセットできるように

kei-nishikawa48 committed:
:fix: レスポンシブのUI修正
- レスポンシブ時tabでcodeかpreviewを切り替えらるように
- apikeyが空でもセットできるように

notozeki reviewed (changes requested):
対応ありがとうございます！レスポンシブとてもいい感じです:+1:　コードの反映がワンテンポ遅れるのも直ってますね！

3点修正をお願いしたいです:pray:

1. APIキーの入力欄の下に、昨日Discordに濱口さんが貼られてたメッセージの表示をお願いします！
2. ↑との関連で、[GPT] に切り替えたときは [Set] を押すまでリクエストが飛ばないようにしたいです（ユーザーのキーを使う場合でも、その前に一回内部キーでリクエストが走ってしまうため）。やりかたは /test ページの実装を参考にしてみてください！
3. 細かいですが、APIキー入力欄のラベルを `apiKey` から `OpenAI API key` に変えてほしいです（何のAPIキーを入力するのかを明確にするため）。

kei-nishikawa48 committed:
:feat: reviewコメントの修正

- 濱口さんのメッセージを表示させるように
apikeyがセットされる前とapikeyが空でセットされた場合はメッセージがのこるように

- setを押すまでリクエストが飛ばないように
- apikeyのラベル修正

notozeki reviewed (approved):
LGTM!
